src/OrchardCore.Modules/OrchardCore.Contents/Controllers/ApiController.cs

```diff
@@ -1,11 +1,13 @@
-﻿using Microsoft.AspNetCore.Authorization;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using OrchardCore.ContentManagement;
 using OrchardCore.Contents;
-using System.Threading.Tasks;
 
 namespace OrchardCore.Content.Controllers
 {
+    [Route("api/content")]
```

Comment:
Curious: should API controller be decorated with `[ApiController]` to use MVC's default conventions for APIs?